miso-web/src/main/webapp/scripts/library_hot.js

```diff
@@ -619,7 +619,7 @@ Library.hot = {
       if (obj.volume) {
         lib.volume = obj.volume;
       } else {
-        lib.volume = 0;
+        lib.volume = null;
```

Comment:
If obj.volume is 0 here, it ends up getting set to null. 0 should be a savable value as well though